### ISSUE TITLE
Add an option to popovers to allow autodismiss

### ISSUE THREAD

**Issue by Blackbaud-ToddRoberts** [blackbaud/skyux-popovers#13]
Add an option to have popovers autodismiss after 3 seconds. This is needed to support the copy to clipboard design #1359

Approved
Designer: @Blackbaud-AdamFunderburk

**Comment by Blackbaud-AlexKingman** [comment]
Closing in favor of #92
